bundler/spec/fixtures/projects/vendored_gems/Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
Nit: when I started hacking on this, my mind exploded wondering why changes to this file aren't picked up by the test harness.

AFAICT the harness overwrites this with https://github.com/dependabot/dependabot-core/blob/main/bundler/spec/fixtures/ruby/gemfiles/Gemfile ?
Maybe a comment that this is only here to make maintaining `vendor/cache` easier? Or remove it?

Reply:
I don't think this gets overwritten, but we do still load the Gemfile from that old location, will update the tests to fix that!